platform-sdk/swirlds-platform-core/src/main/java/module-info.java

```diff
@@ -71,12 +71,7 @@
             com.swirlds.platform.test,
             com.swirlds.platform.core.test.fixtures,
             com.hedera.node.app;
-    exports com.swirlds.platform.crypto to
-            com.swirlds.platform.test,
-            com.hedera.node.test.clients,
-            com.swirlds.platform.core.test.fixtures,
```

Comment:
I'm looking at the delta from develop for all commits now that we've removed platform level changes.   There are no more platform owned code changes except this module file.   Is there a reason these changes are needed?  Or can they be reverted?

Reply:
Checking this now, thanks.